controllers/replication-controller/dellcsireplicationgroup_controller.go

```diff
@@ -56,6 +56,7 @@ type ReplicationGroupReconciler struct {
 	EventRecorder      record.EventRecorder
 	PVCRequeueInterval time.Duration
 	Config             connection.MultiClusterClient
+	ReplicationClient  *csireplication.Replication
```

Comment:
May I know why is this added here?

Reply:
+1. We shouldn't need to add the replication client to this controller for a unit test.

Reply:
I have updated that